RELEASING.md

```diff
@@ -32,9 +32,6 @@ To release a new version of **stactools** or a [stactools-package](https://githu
    Include a "Release summary" section which includes the contents of your section of the CHANGELOG.
 9. Once approved, merge the branch.
 10. Create an **annotated** tag at the new **main** HEAD named `vX.X.X`, e.g. `v0.3.0`.
-    - The contents of the tag should be the CHANGELOG contents for your version release.
-      Be sure to remove any leading `###`, they'll be considered comments in the tag contents.
-      See previous annotated tags for examples of formatting.
```

Comment:
What are the recommended tag contents now?

Reply:
There aren't any -- I used to put stuff in the annotated tags, but that was before CHANGELOGs were more common. Now, I don't think it's worth duplicating the CHANGELOG content in the annotated tags -- folks don't know to look in the tags usually, anyways.